src/index.js

```diff
@@ -34,7 +34,7 @@ if (module.hot) {
       <AppContainer>
         <NextApp store={store} history={syncedHistory} />
       </AppContainer>
-      ), rootElement
+      ), rootElement,
```

Comment:
hmm cool!

the code update by myself.

and i use `eslint src --fix`. the result is same with me.